image.yaml

```diff
@@ -53,5 +55,5 @@ run:
           - "0.0.0.0"
 osbs:
       repository:
-            name: jboss-datagrid-7-docker
-            branch: jb-datagrid71
+            name: containers/jboss-datagrid-7
+            branch: jb-datagrid-7.1-rhel-7
```

Comment:
Is this correct?

Reply:
Yes, we have switched to the new containers namespace for the standalone image, so all branches (ga, -dev and -cp) should use it. Also, when building from -dev or -cp, I'm using an overrides.yaml file (to select the proper -dev or -cp branch), I shall open PRs to add them to our repo next.

Reply:
Here's the reasoning: https://docs.engineering.redhat.com/display/CE/2018/03/14/Container+namespace+and+standalone+images

Reply:
Thanks, just verifying.